flasksrc/quiz.py

```diff
@@ -58,3 +84,76 @@ def get(self, quiz_id):
                 )
 
         return {'items': result}, 200
+
+
+class ToyNetQuizScore(MethodResource):
+    @jwt_required()
+    @use_kwargs(ToyNetQuizScorePostReq)
+    @marshal_with(ToyNetQuizScorePostResp)
+    def post(self, **kwargs):
+        try:
+            req = ToyNetQuizScorePostReq().load(kwargs)
+        except ValidationError as e:
+            abort(400, message=f'malformed create submission request: {e.messages}')
+
+        quiz_id = req['quiz_id']
+        user_id = req['user_id']
+        count_correct = req['count_correct']
+        count_wrong = req['count_wrong']
+
+        db = get_db()
+        try:
+            db.execute(
+                'INSERT INTO toynet_quiz_scores(quiz_id,user_id,count_correct,count_wrong)'
+                ' VALUES((?), (?), (?), (?))',
+                (quiz_id, user_id, count_correct, count_wrong)
+            )
+            db.commit()
+            rows = db.execute(
+                'SELECT scores.submission_id, scores.submitted'
+                ' FROM toynet_quiz_scores AS scores'
+                ' WHERE scores.quiz_id = (?) AND scores.user_id = (?)'
+                ' ORDER BY scores.submitted DESC',
```

Comment:
I'm not quite sure this is true, but for your failing test, it's possible that the `scores.submitted DESC` does not give you the latest score first. It might be worth printing all the results before `return` statement to see in what order the scores are returned!